components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/internal/WorkflowImplServiceComponent.java

```diff
@@ -168,11 +173,15 @@ private void addDefaultBPSProfile() {
                 bpsProfileDTO.setCallbackUser(userName);
                 bpsProfileDTO.setCallbackPassword(password);
                 bpsProfileDTO.setProfileName(WFConstant.DEFAULT_BPS_PROFILE);
-
-                workflowImplService.addBPSProfile(bpsProfileDTO, MultitenantConstants.SUPER_TENANT_ID);
-                if (log.isDebugEnabled()) {
-                    log.info("Default BPS profile added to the DB");
+                if (currentBpsProfile == null) {
+                    workflowImplService.addBPSProfile(bpsProfileDTO, MultitenantConstants.SUPER_TENANT_ID);
+                    if (log.isDebugEnabled()) {
+                        log.info("Default BPS profile added to the DB");
```

Comment:
Its an info. No need to check isDebugEnabled(). Already Fixed with https://github.com/wso2/carbon-identity/commit/006e174d7984fd51fc8840dbd1af0f869505122b.